### ISSUE TITLE
Option to disable pagination

### ISSUE THREAD

**Issue by maloddon** [dchester/epilogue#38]
It might be interesting to have an option to disable pagination, an retrieve all objects (instead of the 1000 harcoded limit), for each resource.

For example : 
```javascript
var resource = epilogue.resource({
	    model: models.categories,
	    endpoints: ['/category', '/category/:id'],
	    actions: ['create', 'list', 'read', 'update'],
	    pagination: false
});
```

**Comment by mbroadst** [comment]
@maloddon I think that's reasonable, pop in a PR and I think we can get that in. Thanks for your interest!

**Comment by mbroadst** [comment]
@maloddon merged with 64c7fac8a37d7e93e45acdba198a1fbd08810e12
